### ISSUE TITLE
[Accessibility]: Code Insights Creation UI - Disable loading announcement for screen readers

### ISSUE THREAD

**Issue by unclejustin** [sourcegraph/sourcegraph#41684]
### Audit type

Screen reader navigation

### Problem description

The preview chart announces "loading" every time a change is made, as well as on the initial page load. This announcement often interrupts other texts.

### Expected behavior

Don't announce loading at all.

I don't think announcing the chart loading is helpful for screen reader users. To have the chart described to them, they have to tab over to it. By that time, the chart is probably finished loading. If it is not, that would be a good time to announce the loading status.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

/cc @joelkw @felixfbecker @vovakulikov

**Comment by coury-clark** [comment]
p1
